src/simple_tracker_launch/config/params-mike.yaml

```diff
@@ -1,3 +1,21 @@
+usb_cam:
```

Comment:
These seem identical to the previous file. Do you imagine all users will have custom params?

Reply:
Not sure, I have only recently included that package, I think it will need to be configured appropriately once we have the allskycam in place. Until then I have just used whatever works for me. I am unsure if the usb_cam package will be used long term, all I was trying to do was, rather than me providing a mechanism for publishing camera images, I thought it would be better to use a community package. This is all part of me trying to standardise things more than they currently are. I don't fully understand all the ins and outs of the package, specifically the camera info file, not sure what that does at the moment, but I wanted to get it included in the tracker and also as part of the launch file.